UIComponents/src/main/java/com/braintreepayments/api/uicomponents/PayPalButtonStyle.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api.uicomponents
+
+import androidx.annotation.ColorInt
+import androidx.core.graphics.toColorInt
+
+enum class PayPalButtonStyle(
```

Comment:
If this is public we should have a docstring. We should also align iOS and Android on what we call this.